packages/wonder-blocks-dropdown/components/select-opener.js

```diff
@@ -222,13 +217,12 @@ const _generateStyles = (light, hovered, focused, pressed) => {
             active: {
                 background: mix(fade(blue, 0.32), white),
                 borderColor: mix(offBlack32, blue),
-                borderStyle: "solid",
-                borderWidth: 1,
+                borderWidth: 2,
+                paddingLeft: 15,
```

Comment:
🤷‍♂️ FWIW, both here and above it seems like it'd be better to use our standard spacing and then do `- 1` (and including the comment above). Not high priority but makes the number a bit clearer.